[2020-06-18 09:52:43,907] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:15:00+00:00 [queued]>
[2020-06-18 09:52:43,915] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:15:00+00:00 [queued]>
[2020-06-18 09:52:43,915] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 09:52:43,915] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-18 09:52:43,915] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 09:52:43,925] {taskinstance.py:900} INFO - Executing <Task(SlackWebhookOperator): slack_msg> on 2018-08-11T00:15:00+00:00
[2020-06-18 09:52:43,927] {standard_task_runner.py:53} INFO - Started process 11073 to run task
[2020-06-18 09:52:43,987] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: slack_example.slack_msg 2018-08-11T00:15:00+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-18 09:52:44,020] {logging_mixin.py:112} INFO - [2020-06-18 09:52:44,019] {base_hook.py:87} INFO - Using connection to: id: anitha_slack. Host: https://hooks.slack.com/services, Port: None, Schema: None, Login: None, Password: XXXXXXXX, extra: None
[2020-06-18 09:52:44,025] {logging_mixin.py:112} INFO - [2020-06-18 09:52:44,025] {base_hook.py:87} INFO - Using connection to: id: anitha_slack. Host: https://hooks.slack.com/services, Port: None, Schema: None, Login: None, Password: XXXXXXXX, extra: None
[2020-06-18 09:52:44,027] {logging_mixin.py:112} INFO - [2020-06-18 09:52:44,027] {http_hook.py:136} INFO - Sending 'POST' to url: https://hooks.slack.com/services
[2020-06-18 09:52:44,038] {logging_mixin.py:112} WARNING - /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
[2020-06-18 09:52:44,069] {logging_mixin.py:112} WARNING - /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
[2020-06-18 09:52:44,124] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=slack_example, task_id=slack_msg, execution_date=20180811T001500, start_date=20200618T095243, end_date=20200618T095244
[2020-06-18 09:52:53,903] {logging_mixin.py:112} INFO - [2020-06-18 09:52:53,903] {local_task_job.py:103} INFO - Task exited with return code 0
[2020-06-18 12:33:57,992] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:15:00+00:00 [queued]>
[2020-06-18 12:33:58,000] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:15:00+00:00 [queued]>
[2020-06-18 12:33:58,000] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 12:33:58,001] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-18 12:33:58,001] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 12:33:58,010] {taskinstance.py:900} INFO - Executing <Task(SlackWebhookOperator): slack_msg> on 2018-08-11T00:15:00+00:00
[2020-06-18 12:33:58,012] {standard_task_runner.py:53} INFO - Started process 6085 to run task
[2020-06-18 12:33:58,070] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: slack_example.slack_msg 2018-08-11T00:15:00+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-18 12:33:58,091] {logging_mixin.py:112} INFO - [2020-06-18 12:33:58,091] {base_hook.py:87} INFO - Using connection to: id: anitha_slack. Host: https://hooks.slack.com/services, Port: None, Schema: None, Login: None, Password: XXXXXXXX, extra: None
[2020-06-18 12:33:58,092] {logging_mixin.py:112} INFO - [2020-06-18 12:33:58,092] {http_hook.py:136} INFO - Sending 'POST' to url: https://hooks.slack.com/services/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
[2020-06-18 12:33:58,102] {logging_mixin.py:112} WARNING - /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
[2020-06-18 12:33:58,134] {logging_mixin.py:112} INFO - [2020-06-18 12:33:58,134] {http_hook.py:150} ERROR - HTTP error: Bad Request
[2020-06-18 12:33:58,134] {logging_mixin.py:112} INFO - [2020-06-18 12:33:58,134] {http_hook.py:151} ERROR - no_text
[2020-06-18 12:33:58,138] {taskinstance.py:1145} ERROR - 400:Bad Request
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/lib/python2.7/site-packages/airflow/contrib/operators/slack_webhook_operator.py", line 113, in execute
    self.hook.execute()
  File "/usr/lib/python2.7/site-packages/airflow/contrib/hooks/slack_webhook_hook.py", line 152, in execute
    extra_options={'proxies': proxies})
  File "/usr/lib/python2.7/site-packages/airflow/hooks/http_hook.py", line 137, in run
    return self.run_and_check(session, prepped_request, extra_options)
  File "/usr/lib/python2.7/site-packages/airflow/hooks/http_hook.py", line 181, in run_and_check
    self.check_response(response)
  File "/usr/lib/python2.7/site-packages/airflow/hooks/http_hook.py", line 152, in check_response
    raise AirflowException(str(response.status_code) + ":" + response.reason)
AirflowException: 400:Bad Request
[2020-06-18 12:33:58,139] {taskinstance.py:1202} INFO - Marking task as FAILED.dag_id=slack_example, task_id=slack_msg, execution_date=20180811T001500, start_date=20200618T123357, end_date=20200618T123358
[2020-06-18 12:34:07,990] {logging_mixin.py:112} INFO - [2020-06-18 12:34:07,990] {local_task_job.py:103} INFO - Task exited with return code 1
